src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,41 @@
+package duke.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this(description, false);
+    }
+
+    public Task(String description, Boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    /**
```

Comment:
The javadoc phrasing for `getStatusIcon` here is a little inappropriate and redundant since your function name is pretty explicit already.